docs/src/rules/consistent-this.md

```diff
@@ -74,7 +74,7 @@ Examples of **incorrect** code for this rule with the default `"that"` option, i
 ```js
 /*eslint consistent-this: ["error", "that"]*/
 
-var that;
+let that;
 function f() {
     that = this;
```

Comment:
this supposed to be an incorrect example but there is no error can you check this?